src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_help.py

```diff
@@ -8,6 +8,25 @@
 helps['iot'] = """
     type: group
     short-summary: (PREVIEW) Manage Internet of Things (IoT) assets.
+    
+    long-summary: 'Welcome to Azure CLI 2.0 IoT documentation! 
```

Comment:
Use YAML string folding syntax rather than quotes here for multi-line documentation:

```
long-summary: >
  Welcome to Azure CLI 2.0 IoT documentation! ...
```

Reply:
Also, I will note this doesn't seem to be an appropriate long summary. No other service does this AFAIK.